reference/configuration/framework.rst

```diff
@@ -2191,7 +2191,7 @@ By default, the :doc:`NotCompromisedPassword </reference/constraints/NotCompromi
 constraint uses the public API provided by `haveibeenpwned.com`_. This option
 allows to define a different, but compatible, API endpoint to make the password
 checks. It's useful for example when the Symfony application is run in an
-intranet without public access to Internet.
+intranet without public access to the Internet.
```

Comment:
```suggestion
intranet without public access to the internet.
```
?

Reply:
It's commonly capitalized: https://en.wikipedia.org/wiki/Internet, but both could be correct.